src/main/java/duke/Task.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Task class that is used by duke class to create tasks.
+ * Stores information like description, task type and whether if it is done.
+ * Stores date if the task is a deadline or event.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String taskType;
```

Comment:
Do you need to store a different instance of the same String in every object for task type? It is better to store it as a static field and override getTaskType() in each subclass.